elixir/apps/domain/lib/domain/cluster/google_compute_labels_strategy.ex

```diff
@@ -132,13 +132,15 @@ defmodule Domain.Cluster.GoogleComputeLabelsStrategy do
 
       {:error, reason} ->
         if remaining_retry_count == 0 do
-          Logger.error("Can't fetch list of nodes or access token: #{inspect(reason)}",
```

Comment:
Moving the error inspection out of the string here will allow Sentry to merge these.